grails-app/services/com/unifina/service/EthereumIntegrationKeyService.groovy

```diff
@@ -141,7 +141,7 @@ class EthereumIntegrationKeyService {
 
 	SecUser createEthereumUser(String address) {
 		SecUser user = userService.createUser([
-			username       : address,
+			email          : address + "@streamr.network",
```

Comment:
Should we save the eth account to it's own db field?

This thing works because email field is unique ATM, otherwise it could be: "anonymous.user@streamr.network"